odbc/src/api/data_tests.rs

```diff
@@ -761,26 +777,13 @@ mod unit {
                     ],
                     SqlReturn::SUCCESS,
                 );
-                bin_val_test(DOC_COL, &[], SqlReturn::ERROR);
-                let errors_len = (*stmt_handle)
-                    .as_statement()
-                    .unwrap()
-                    .errors
-                    .read()
-                    .unwrap()
-                    .len();
-                assert_eq!(
-                    "[MongoDB][API] BSON type object cannot be converted to ODBC type Binary"
-                        .to_string(),
-                    format!(
-                        "{}",
-                        (*stmt_handle)
-                            .as_statement()
-                            .unwrap()
-                            .errors
-                            .read()
-                            .unwrap()[errors_len - 1]
-                    ),
+                bin_val_test(
+                    DOC_COL,
+                    &[
+                        123, 32, 34, 120, 34, 58, 32, 52, 50, 44, 32, 34, 121, 34, 58, 32, 52, 50,
```

Comment:
I have one suggestion for these tests.  Now that we have binary values that are readable.  How about we modify these tests to have the expected value be the string with .as_bytes().  That would make it more readable, such as: `"{ \"x\": 42, \"y\": 42 }".as_bytes()".  See if it works, otherwise we can stick with how it is written.

Reply:
great suggestion, I'll work on it